src/chrome/content/rules/First_Look.org.xml

```diff
@@ -1,4 +1,12 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://prod01-cdn00.cdn.firstlook.org/ => https://prod01-cdn00.cdn.firstlook.org/: Too many redirects while fetching 'https://prod01-cdn00.cdn.firstlook.org/'
+Fetch error: http://prod01-cdn01.cdn.firstlook.org/ => https://prod01-cdn01.cdn.firstlook.org/: Too many redirects while fetching 'https://prod01-cdn01.cdn.firstlook.org/'
+Fetch error: http://prod01-cdn02.cdn.firstlook.org/ => https://prod01-cdn02.cdn.firstlook.org/: Too many redirects while fetching 'https://prod01-cdn02.cdn.firstlook.org/'
+Fetch error: http://prod01-cdn03.cdn.firstlook.org/ => https://prod01-cdn03.cdn.firstlook.org/: Too many redirects while fetching 'https://prod01-cdn03.cdn.firstlook.org/'
+Fetch error: http://prod01-piwik.firstlook.org/ => https://prod01-piwik.firstlook.org/: (28, 'Connection timed out after 10000 milliseconds')
+
```

Comment:
These loop whether fetched over http or https. Does it really matter--I mean, who's going to encounter these?